engine/src/main/java/org/destinationsol/game/chunk/ChunkFiller.java

```diff
@@ -357,12 +357,12 @@ private void fillDust(SolGame game, Vector2 chunkCenter, RemoveController remove
     /**
      * Find a random position in a chunk centered around chunkCenter, relative to the entire map, and make sure it is not yet
      * occupied by another entity.
-     * <p/>
-     * Up to 100 tries will be made to find an unoccupied position; if by then none has been found, <code>null</code> will be returned.
+     * <p>
+     * Up to 100 tries will be made to find an unoccupied position; if by then none has been found, null will be returned.
      *
      * @param game        The {@link SolGame} instance to work with
      * @param chunkCenter The center of a chunk in which a random position should be found
-     * @return A random, unoccupied position in a chunk centered around chunkCenter, relative to the entire map, or <code>null</code> if within 100 tries no unoccupied position has been found
+     * @return A random, unoccupied position in a chunk centered around chunkCenter, relative to the entire map, or null if none found within 100 tries
```

Comment:
Please revert back to the original wording on this.

As things stand, `within 100 tries` gives the impression that `tries` is a unit. The previous one, although verbose, was clear.

Reply:
Do we even want to include the "100 tries" in here? What about writing just `or null if no unoccupied position has been found`? As far as I have always been concerned, javadocs should consider more of the usage of the method/class instead of implementation details, unless the implementation details are something like `is very slow during full moon, please consider using ... in case of full moon`. Here, the "100 tries" have no added value for the javadocs.

Reply:
Let's go with `after a certain number of unsuccessful tries`?

Reply:
Hey, if you look at the main text of the javadocs, the `@return` looks basically like concatenated version of that. What about this wording: 
`The found unoccupied position, or null if no position has been found.`

Reply:
Okay, works for me.